### PR TITLE
Added nodeSelector for celery deployment and migrate job

### PR DESCRIPTION
**Pull request by pikilisaikiran**
# What this PR does
Adds nodeSelector for celery deployment and migrate job

## Which issue(s) this PR fixes
Fixes errors while deploying resources in a cluster with Gatekeeper policy ( that restricts deployments without nodeSelector).

## Checklist

- [x] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.15 (2023-01-05)
+
+### Added
+
+- Added nodeSelector for celery deployment and migrate job.
```
Comment (joeyorlando) [inline CHANGELOG.md:12]:
```suggestion
- Added nodeSelector for celery deployment and migrate job.

```

File: helm/oncall/values.yaml [hunk 2]
```diff
@@ -148,6 +151,7 @@ oncall:
 # Whether to run django database migrations automatically
 migrate:
   enabled: true
+  nodeSelector: {}
```
Comment (joeyorlando) [inline helm/oncall/values.yaml:154]:
```suggestion
  ## Node labels for pod assignment
  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
  nodeSelector: {}
```

**Review by joeyorlando — COMMENTED** [review]
few small comments:
- we should also add the `nodeSelector` object under the `engine` object in `helm/oncall/values.yaml`, and add that to `helm/oncall/templates/engine/deployment.yaml`
- currently a merge conflict in `CHANGELOG.md`

**Comment by pikilisaikiran** [comment]
- `nodeSelector` is already present for `engine` object
- merge conflict is also resolved in `CHANGELOG.md`

**Pull request merged**